src/code/containers/challenge-container-selector.js

```diff
@@ -24,7 +24,7 @@ function mapContainerNameToContainer(containerName) {
 class ChallengeContainerSelector extends Component {
 
   static propTypes = {
-    authoring: PropTypes.object.isRequired,
+    authoring: PropTypes.array.isRequired,
```

Comment:
This was throwing errors before I changed it- `authoring` *is* an array, so I can see why this is correct, but I don't know why it wasn't a problem when these propTypes were in `challenge-container`